src/app/renderer/components/app/FileBrowser/components/Sidebar/Sidebar.tsx

```diff
@@ -27,12 +27,22 @@ import {If} from '../../../../helper/If';
 import styles from './SidebarStyles.scss';
 import {selectFilter} from '../../../../../actions/fileTree/selectFilter';
 import {resetFilter} from '../../../../../actions/fileTree/resetFilter';
+import {TextField} from '../../../../form/TextField';
+import {searchForFile} from "../../../../../actions/fileTree/searchForFile";
 
 interface Injected {
     projectState: ProjectState;
     editorState: EditorState;
 }
 
+const debounce = (fn: Function, ms: number = 400) => {
```

Comment:
Optional: I think, you can remove the default ms as you will probably always provide it from the caller side